packages/react-devtools-scheduling-profiler/src/view-base/View.js

```diff
@@ -270,19 +288,35 @@ export class View {
   handleInteractionAndPropagateToSubviews(
     interaction: Interaction,
     viewRefs: ViewRefs,
-  ) {
+  ): boolean {
     const {subviews, visibleArea} = this;
 
     if (visibleArea.size.height === 0) {
-      return;
+      return false;
     }
 
-    this.handleInteraction(interaction, viewRefs);
-
-    subviews.forEach(subview => {
+    // Pass the interaction to subviews first,
+    // so they have the opportunity to claim it before it bubbles.
+    for (let i = 0; i < subviews.length; i++) {
```

Comment:
Something I noticed the old code did: I looped through the list of subviews in reverse because later subviews can draw over earlier ones, so we’d want later subviews to handle events first.

https://github.com/MLH-Fellowship/scheduling-profiler-prototype/blob/46997525744263b1614c2b617167ddcb33588ce7/src/layout/StaticLayoutView.js#L83-L85

Reply:
Yeah! I could have sworn my bubbling code did the same but I guess that was an earlier iteration that I didn't end up committing. I'll change it to do this though )